x-pack/plugins/cases/public/components/recent_cases/recent_cases.tsx

```diff
@@ -30,46 +30,66 @@ const MarkdownContainer = styled.div`
-  });
+export const RecentCasesComp = React.memo<RecentCasesProps>(
+  ({ filterOptions, maxCasesToShow, recentCasesFilterBy }) => {
+    const { owner } = useCasesContext();
+    const availableSolutions = useAvailableCasesOwners(['read']);
+    const hasOwner = !!owner.length;
 
-  return isLoadingCases ? (
-    <LoadingPlaceholders lines={2} placeholders={3} />
-  ) : !isLoadingCases && data.cases.length === 0 ? (
-    <NoCases />
-  ) : (
-    <>
-      {data.cases.map((c, i) => (
-        <EuiFlexGroup key={c.id} gutterSize="none" justifyContent="spaceBetween">
-          <EuiFlexItem grow={false}>
-            <EuiText size="s">
-              <CaseDetailsLink detailName={c.id} title={c.title}>
-                <TruncatedText text={c.title} />
-              </CaseDetailsLink>
-            </EuiText>
+    const { data = initialGetCasesData, isLoading: isLoadingCases } = useGetCases({
+      queryParams: { perPage: maxCasesToShow },
+      filterOptions: { ...filterOptions, owner: hasOwner ? owner : availableSolutions },
+    });
 
-            <IconWithCount count={c.totalComment} icon={'editorComment'} tooltip={i18n.COMMENTS} />
-            {c.description && c.description.length && (
-              <MarkdownContainer>
-                <EuiText color="subdued" size="xs">
-                  <MarkdownRenderer disableLinks={true}>{c.description}</MarkdownRenderer>
-                </EuiText>
-              </MarkdownContainer>
-            )}
-            {i !== data.cases.length - 1 && <EuiSpacer size="l" />}
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      ))}
-    </>
-  );
-};
+    return isLoadingCases ? (
+      <LoadingPlaceholders lines={2} placeholders={3} />
+    ) : !isLoadingCases && data.cases.length === 0 ? (
+      <NoCases recentCasesFilterBy={recentCasesFilterBy} />
+    ) : (
+      <>
+        {data.cases.map((c, i) => (
+          <EuiFlexGroup key={c.id} gutterSize="none" justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">
+                <CaseDetailsLink detailName={c.id} title={c.title}>
+                  <TruncatedText text={c.title} />
+                </CaseDetailsLink>
+              </EuiText>
+              <EuiSpacer size="xs" />
+              {c.description && c.description.length && (
+                <MarkdownContainer>
+                  <EuiText color="subdued" size="xs">
+                    <TruncatedText text={c.description} />
+                  </EuiText>
+                </MarkdownContainer>
+              )}
+              <EuiSpacer size="xs" />
+              <EuiFlexGroup gutterSize="s">
+                <EuiFlexItem grow={false}>
+                  <EuiText
+                    size="xs"
+                    color="default"
+                    data-test-subj="recent-cases-creation-relative-time"
+                  >
+                    <UserActionTimestamp createdAt={c.createdAt} />
```

Comment:
We should not be dependent on the user actions in this component. If in the future we change something in the user action timestamp it will affect the recent cases component. It is very easy to forget that. You can just 

```
<LocalizedDateTooltip date={new Date(c.createdAt)}>
      <FormattedRelative
        data-test-subj="recent-cases-time"
        value={createdAt}
      />
</LocalizedDateTooltip>
```

or make it a new component here `x-pack/plugins/cases/public/components/formatted_date/index.tsx` and import it.

Reply:
done